src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.VisualStudio.RazorExtension
     // since we want to have the option to own the experience for Razor files
     [ProvideEditorExtension(typeof(RazorEditorFactory), RazorLSPContentTypeDefinition.CSHTMLFileExtension, 52, NameResourceID = 101)]
     [ProvideEditorExtension(typeof(RazorEditorFactory), RazorLSPContentTypeDefinition.RazorFileExtension, 52, NameResourceID = 101)]
-    [ProvideEditorFactory(typeof(RazorEditorFactory), 101, CommonPhysicalViewAttributes = (int)__VSPHYSICALVIEWATTRIBUTES.PVA_SupportsPreview)]
+    [ProvideEditorFactory(typeof(RazorEditorFactory), 101, deferUntilIntellisenseIsReady: false, CommonPhysicalViewAttributes = (int)__VSPHYSICALVIEWATTRIBUTES.PVA_SupportsPreview)]
```

Comment:
Haha, that is a very specific parameter, but considering it's the exact one we need I guess we can't complain.